cli/demo_sidechain.sh

```diff
@@ -37,23 +37,43 @@ while getopts ":m:p:A:B:t:" opt; do
         B)
             WORKER2PORT=$OPTARG
             ;;
+        u)
+            NODEURL=$OPTARG
+            ;;
+        W)
```

Comment:
Ports for workers are alphabetical (A, B). I would do it likewise with the URL: V, W.

Reply:
Yes, agreed 👍